content/en/news/releases/1.16.x/announcing-1.16/change-notes/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Istio 1.16.0 Change Notes
+linktitle: 1.16.0
+subtitle: Minor Release
+description: Istio 1.16.0 change notes.
+publishdate: 2022-11-15
+release: 1.16.0
+weight: 10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** fetching charts from URLs in `istio-operator`.
+
+## Traffic Management
+
+- **Improved** sidecar `Host` header matching to ignore port numbers by default. This can be controlled by the `SIDECAR_IGNORE_PORT_IN_HOST_MATCH` environment variable. ([Issue #36627](https://github.com/istio/istio/issues/36627))
+
+- **Updated** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces where istiod creates the `istio-ca-root-cert` configmap
+  if the `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled.
+
+- **Updated** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces where istiod discovers Custom Resource configurations
+  (like Gateway, VirtualService, DestinationRule, Ingress, etc.) if the `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled.
+  ([Issue #36627](https://github.com/istio/istio/issues/36627))
+
+- **Updated** the gateway-api integration to read `v1beta1` resources for `HTTPRoute`, `Gateway`, and `GatewayClass`. Users of the gateway-api must
+  be on version 0.5.0+ before upgrading Istio.
```

Comment:
Does this mean user need to update gateway resource first to version 0.5.0+ before upgrading istio?  cc @howardjohn

Reply:
Yes, there is a `istioctl pre-check` for it.

You can also just not have the gateway-api installed at all

Reply:
This should probbly be in upgrade-notes as well

Reply:
Adding this to the upgrade-notes page with a link to the guide Frank merged.